boards/xtensa/m5stickc_plus/m5stickc_plus_defconfig

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_XTENSA_RESET_VECTOR=n
+
+CONFIG_BOARD_M5STICKC_PLUS=y
+CONFIG_SOC_ESP32=y
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=240000000
```

Comment:
You could remove this line as `SYS_CLOCK_HW_CYCLES_PER_SEC` already is set to default to .DTS 240MHz.